bliss.js

```diff
@@ -483,57 +507,43 @@ $.setProps = {
 		}
 	},
 
-	once: function(val) {
-		if (arguments.length == 2) {
-			val = {};
-			val[arguments[0]] = arguments[1];
-		}
-
+	once: overload(function(events, callback){
```

Comment:
events is problematic with overload as it supports an element as a param. An element would pass for not String, and would get iterated on by overload.

IMO events would be a separate effort outside of this initial implementation of overload.